application/src/main/java/org/togetherjava/tjbot/commands/moderation/WhoIsCommand.java

```diff
@@ -0,0 +1,263 @@
+package org.togetherjava.tjbot.commands.moderation;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.GuildVoiceState;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.components.Button;
+import net.dv8tion.jda.api.interactions.components.ButtonStyle;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyAction;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+
+import javax.annotation.CheckReturnValue;
+import java.awt.*;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * You ever wanted to stalk someone on Discord? Well no need to worry anymore! I got you covered,
+ * this command allows you to look up user (or member) info.
+ */
+@SuppressWarnings("ClassWithoutLogger")
+public final class WhoIsCommand extends SlashCommandAdapter {
+    private static final String USER_OPTION = "user";
+    private static final String SHOW_SERVER_SPECIFIC_INFO = "show_server_specific_info";
```

Comment:
The constant name should hint what it is, i.e. an option or a subcommand or something else. Maybe `SHOW_SERVER_INFO_OPTION`